README.md

```diff
@@ -40,24 +46,85 @@ If you intend to use the software constantly, you can install for the user or at
 Installation for the user can be carried out by:
 
 ```bash
-$ cp h1_client.bin ~/.local/bin/h1
+$ cp h1-linux.bin ~/.local/bin/h1
 $ h1
 ```
 
 Installation at the system level can be carried out by:
 
 ```bash
-$ cp h1_client.bin /usr/local/bin/h1
+$ cp h1-linux.bin /usr/local/bin/h1
+$ h1
+```
+
+Operation might be not permitted and you might need to execute it in sudo mode:
+
+```bash
+$ sudo cp ./h1-linux.bin /bin/h1
 $ h1
 ```
 
 ### Windows environment
 
-> Introduction for Windows requires development taking into account the specificity of the platform.
+Download the *h1-win.exe* file from [Releases], then run the downloaded executable file using CMD or Powershell:
```

Comment:
how?

Reply:
Check last commit, I explain how to download it using Powershell